### ISSUE TITLE
the hyper-parameter of the pretrain on CIFAR_FS and FC100

### ISSUE THREAD

**Issue by KZF-kzf** [zhangdoudou123/SemFew#9]
hi,i want to know the hyper-parameter of the pretrain on CIFAR_FS and FC100, and the accuracy of your pretrain weight in val dataset.

**Comment by linjieb** [comment]
> hi,i want to know the hyper-parameter of the pretrain on CIFAR_FS and FC100, and the accuracy of your pretrain weight in val dataset.

Hi,have you finished the repetition of the paper?I have done it,but the result is bad.If so,could you please talk about it with me?
